test/integration/integration_suite_test.go

```diff
@@ -207,17 +226,30 @@ var _ = AfterSuite(func(done Done) {
 	var err error
 	for _, cluster := range clusters {
 		kubeCtl.SetContext(cluster)
+		cl := k8sClients[cluster]
 		if len(cluster) == 0 {
 			cluster = "default"
 		}
 		By(cluster + ": Check for controllers' restarts")
 		err = utils.CheckRestart(kubeCtl)
 		if err != nil {
-			logf.Log.Error(err, "Error restarting nephe controller")
+			logf.Log.Error(err, "error restarting nephe controller")
 			cl := cluster
 			controllersCored = &cl
 			break
 		}
+		if withAgent {
+			By(fmt.Sprintf(cluster+": Deleting namespace %s", staticVMNS.Name))
+			err = cl.Delete(context.TODO(), staticVMNS)
+			Expect(err).ToNot(HaveOccurred())
+			dir, err := os.UserHomeDir()
+			if err != nil {
+				logf.Log.Error(err, "error getting home directory")
+				return
+			}
+			dir = dir + "/tmp/integration/"
```

Comment:
can we declare a constant for it?

Reply:
Using Rahul's suggestion, creating a random suffixed tmp dir with build in functions.